xbmc/video/VideoDatabase.cpp

```diff
@@ -12618,12 +12618,15 @@ void CVideoDatabase::AddVideoAsset(VideoDbContentType itemType,
     m_pDS->query(PrepareSQL("SELECT idFile FROM videoversion WHERE idFile = %i", idFile));
 
     if (m_pDS->num_rows() == 0)
-      m_pDS->exec(PrepareSQL("INSERT INTO videoversion VALUES(%i, %i, '%s', %i, %i)", idFile, dbId,
-                             mediaType.c_str(), videoAssetType, idVideoVersion));
+      m_pDS->exec(
+          PrepareSQL("INSERT INTO videoversion (idFile, idMedia, media_type, itemType, idType)"
```

Comment:
Shouldn't there be a space at the end of this string? Not sure if it's relevant.